rules/windows/network_connection/net_connection_win_regasm_network_activity.yml

```diff
@@ -0,0 +1,34 @@
+title: Network Connection Initiated By RegAsm.EXE
+id: 0531e43a-d77d-47c2-b89f-5fe50321c805
+status: experimental
+description: Detects a network connection initiated by "RegAsm.exe"
+references:
+    - https://app.any.run/tasks/ec207948-4916-47eb-a0f4-4c6abb2e7668/
+    - https://research.splunk.com/endpoint/07921114-6db4-4e2e-ae58-3ea8a52ae93f/
+author: frack113
+date: 2024/04/18
+tags:
+    - attack.defense_evasion
+    - attack.t1218.009
+logsource:
+    category: network_connection
+    product: windows
+detection:
+    selection:
+        Initiated: 'true'
+        Image|endswith: '\regasm.exe'
+        DestinationPort: 587 # SMTP
```

Comment:
Why on this port?

Reply:
This port was from the AgentTesla in the sandbox.
As I can check FP , I have limited the `DestinationPort` to keep the rule high.
A medium without it is perhaps a better way.

Reply:
Would need to check some more FP. Before we can merge this as the rule is not very useful at the moment.

Reply:
Checked out some resources. It seems one of the most common is port 80 and even on some VT results. I will reduce it to medium for now. Once its merged i'll give it a week and check VT results for more accuracy.